modern/clang/Dockerfile

```diff
@@ -0,0 +1,149 @@
+ARG DOCKER_USERNAME
+
+FROM ubuntu:xenial as builder
+
+ARG LLVM_VERSION ARTIFACTORY_REPOSITORY LIBSTDCPP_VERSION
+
+LABEL maintainer="Conan.io <info@conan.io>"
+
+# FIXME: Clang 9 and 10 fail to build libcxx because they try to link libc++abi from system.
+#        We could add a patch with target_link_directories, but I prefered copying libc++abi to /usr/lib
```

Comment:
Is this comment true for Clang 10? I think we need to update thing `FIXME`.

Reply:
Yes, is true. At line 112 there a copy which fixes it.

Reply:
I'll keep Clang 9 listed as info. As you can see, each llvm version has a different behavior.